cfme/common/provider.py

```diff
@@ -532,8 +536,10 @@ def delete(self, cancel=True):
             cancel: Whether to cancel the deletion, defaults to True
         """
         view = navigate_to(self, 'Details')
-        item_title = version.pick({'5.9': 'Remove this {} Provider from Inventory',
-                                   version.LOWEST: 'Remove this {} Provider'})
+        item_title = VersionPicker({
```

Comment:
**required** there seem to be inconsistencies in picking 5.9 vs lowest, some being removed, some being added - we should have this uniformly